server/src/main.rs

```diff
@@ -190,8 +190,10 @@ fn main() {
 
   // Start chat server actor in separate thread
   let server = ChatServer::default().start();
-  // Create Http server with websocket support
 
+  let settings = lemmy_server::Settings::get();
```

Comment:
Instead of `lemmy_server::Settings`, do an import like those at the top, so you can just do `Settings::`

Reply:
Done